src/Ddeboer/DataImport/Filter/DateTimeThresholdFilter.php

```diff
@@ -47,14 +47,14 @@ public function __construct(
     /**
      * {@inheritDoc}
      */
-    public function filter(array $item)
+    public function __invoke(array $item)
```

Comment:
Just checking: what exactly are the arguments for using `__invoke` over interfaces? Only that using external libraries (such as Zend) will be easier?